frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -52,6 +54,21 @@
     return a.user.name.localeCompare(b.user.name);
   });
 
+  let lexEntryCount = derived(projectStore, project => {
```

Comment:
instead of doing this we should just add the field to the gql query. If there's no value or it hasn't been updated yet that's fine.

Reply:
I just pushed a commit moving the fetch into the load function, but not in the gql query. I thought about putting it in the gql query, but I wanted it to be created on-demand if the value hadn't been calculated yet, and wasn't sure how to do that. You may want to look at commit db398f8 and review that one.

Reply:
Done. It's in the GQL query now, and we fetch the value from the API only if we get null (or undefined) from the query AND it's a FLEx project.